libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -173,16 +203,29 @@ DNNModel *ff_dnn_load_model_ov(const char *model_filename, const char *options)
     if (!ov_model)
         goto err;
     ov_model->ctx.class = &dnn_openvino_class;
+    ctx = &ov_model->ctx;
```

Comment:
why we need this?

Reply:
since we need the ctx to show log in following av_log()
which have been missed the the "dnn/openvino: add error log" patch

Reply:
ok